src/Microsoft.Data.SqlClient/tests/Directory.Build.props

```diff
@@ -12,6 +12,7 @@
     <Configurations>Debug;Release;</Configurations>
     <Platforms>AnyCPU;x86;x64</Platforms>
     <ReferenceType Condition="'$(ReferenceType)'==''">Project</ReferenceType>
+    <NuGetAudit>false</NuGetAudit>
```

Comment:
To prevent NuGet audit on test projects giving warnings as errors on old packages